.travis.yml

```diff
@@ -16,5 +16,5 @@ script:
   - npm run test
   - npm run clean-backups
 before_script:
-  - ganache-cli -b 1 -m "shove afford modify census bridge good random error fault floor fringe oblige" -i 1002 > /dev/null &
+  - ganache-cli -m "shove afford modify census bridge good random error fault floor fringe oblige" -i 1002 > /dev/null &
```

Comment:
I know that automine is faster, but I would leave the blocktime there if possible, to make it as close to a real-life blockchain. Unless there is an issue with using blocktime.

Reply:
The issue with block time is we would need to wait 255 seconds before starting tests when claim window is 255. We would need to use different claim window size. 

From one side there's an obvious benefit of testing interval based timenode mechanics in waiting for blocks (with automine disabled), on other hand we have more deterministic system with lesser time involvement.